cli/data.go

```diff
@@ -261,7 +261,9 @@ func createDataFilter(c *cli.Context) (*datapb.Filter, error) {
 	return filter, nil
 }
 
-// BinaryData downloads binary data matching filter to dst.
+// Download binary data matching provided filter to destination directory.
+//    // filter := createDataFilter(TODO)
```

Comment:
@benjirewis I tried to dig into how to create a filter for about 15 minutes but decided to just ask - could you help me with this? I think it's something with `cctx` but I'm not sure whether you're supposed to create it like this at all.